templates/go-carbon.conf.erb

```diff
@@ -24,38 +23,163 @@ aggregation-file = "<%= @whisper_aggregation_file %>"
+tagdb-url = "http://127.0.0.1:8000"
+tagdb-chunk-size = 32
+tagdb-update-interval = 100
+# Directory for send queue (based on leveldb)
+local-dir = "/var/lib/graphite/tagging/"
+# POST timeout
+tagdb-timeout = "1s"
+
+[carbonserver]
+# Please NOTE: carbonserver is not intended to fully replace graphite-web
+# It acts as a "REMOTE_STORAGE" for graphite-web or carbonzipper/carbonapi
+listen = "127.0.0.1:8080"
+# Carbonserver support is still experimental and may contain bugs
+# Or be incompatible with github.com/grobian/carbonserver
+enabled = false
+# Buckets to track response times
+buckets = 10
+# carbonserver-specific metrics will be sent as counters
+# For compatibility with grobian/carbonserver
+metrics-as-counters = false
+# Read and Write timeouts for HTTP server
+read-timeout = "60s"
+write-timeout = "60s"
+# Enable /render cache, it will cache the result for 1 minute
+query-cache-enabled = true
+# 0 for unlimited
+query-cache-size-mb = 0
+# Enable /metrics/find cache, it will cache the result for 5 minutes
+find-cache-enabled = true
+# Control trigram index
+#  This index is used to speed-up /find requests
+#  However, it will lead to increased memory consumption
+#  Estimated memory consumption is approx. 500 bytes per each metric on disk
+#  Another drawback is that it will recreate index every scan-frequency interval
+#  All new/deleted metrics will still be searchable until index is recreated
+trigram-index = true
+# carbonserver keeps track of all available whisper files
+# in memory. This determines how often it will check FS
+# for new or deleted metrics.
+scan-frequency = "5m0s"
+# Maximum amount of globs in a single metric in index
+# This value is used to speed-up /find requests with
+# a lot of globs, but will lead to increased memory consumption
+max-globs = 100
+# Fail if amount of globs more than max-globs
+fail-on-max-globs = false
+# graphite-web-10-mode
+# Use Graphite-web 1.0 native structs for pickle response
+# This mode will break compatibility with graphite-web 0.9.x
+# If false, carbonserver won't send graphite-web 1.0 specific structs
+# That might degrade performance of the cluster
+# But will be compatible with both graphite-web 1.0 and 0.9.x
+graphite-web-10-strict-mode = true
+# Allows to keep track for "last time readed" between restarts, leave empty to disable
+internal-stats-dir = ""
+# Calculate /render request time percentiles for the bucket, '95' means calculate 95th Percentile. To disable this feature, leave the list blank
+stats-percentiles = [99, 98, 95, 75, 50]
+
+[dump]
+# Enable dump/restore function on USR2 signal
+enabled = false
+# Directory for store dump data. Should be writeable for carbon
+path = "/var/lib/graphite/dump/"
+# Restore speed. 0 - unlimited
+restore-per-second = 0
 
 [pprof]
 listen = "<%= @pprof_listen %>"
-enabled = <%= @pprof_enabled %>
\ No newline at end of file
+enabled = <%= @pprof_enabled %>
+
+# Default logger
+[[logging]]
```

Comment:
Is the section title supposed to be nested for logging?

Reply:
Looks like it, based off the go-carbon.conf generated from the 0.13.0 deb install and current README. Logging appears to be functioning correctly in vagrant